lib/fastlane_core/devices.rb

```diff
@@ -1,14 +1,19 @@
 require 'open3'
 
 module FastlaneCore
-  class Simulator
+  class Devices
```

Comment:
I'm not sure this is the best name for this class. I definitely understand the desire to rename it from `Simulator` but something feels a bit weird with having a plural name for a class that we already have and it is not just a collection of `Device` instances.

Maybe this can be something like `DeviceController` or `DeviceManager`, frankly, I'm not sure I like either of those options I just suggested, but as they say, 'naming things is hard'  :smile:. Would love to hear your thoughts. :octopus: